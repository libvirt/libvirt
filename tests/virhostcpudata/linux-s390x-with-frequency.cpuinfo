vendor_id       : IBM/S390
# processors    : 8
bogomips per cpu: 20325.00
max thread id   : 1
features	: esan3 zarch stfle msa ldisp eimm dfp edat etf3eh highgprs te vx sie
facilities      : 0 1 2 3 4 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 30 31 32 33 34 35 36 37 40 41 42 43 44 45 46 47 48 49 50 51 52 53 55 57 64 65 66 67 68 69 70 71 72 73 75 76 77 78 80 128 129 131 132 142 143
cache0          : level=1 type=Data scope=Private size=128K line_size=256 associativity=8
cache1          : level=1 type=Instruction scope=Private size=96K line_size=256 associativity=6
cache2          : level=2 type=Data scope=Private size=2048K line_size=256 associativity=8
cache3          : level=2 type=Instruction scope=Private size=2048K line_size=256 associativity=8
cache4          : level=3 type=Unified scope=Shared size=65536K line_size=256 associativity=16
cache5          : level=4 type=Unified scope=Shared size=491520K line_size=256 associativity=30
processor 0: version = 00,  identification = 145F07,  machine = 2964
processor 1: version = 00,  identification = 145F07,  machine = 2964
processor 2: version = 00,  identification = 145F07,  machine = 2964
processor 3: version = 00,  identification = 145F07,  machine = 2964
processor 4: version = 00,  identification = 145F07,  machine = 2964
processor 5: version = 00,  identification = 145F07,  machine = 2964
processor 6: version = 00,  identification = 145F07,  machine = 2964
processor 7: version = 00,  identification = 145F07,  machine = 2964

cpu number      : 0
cpu MHz dynamic : 5000
cpu MHz static  : 5000

cpu number      : 1
cpu MHz dynamic : 5000
cpu MHz static  : 5000

cpu number      : 2
cpu MHz dynamic : 5000
cpu MHz static  : 5000

cpu number      : 3
cpu MHz dynamic : 5000
cpu MHz static  : 5000

cpu number      : 4
cpu MHz dynamic : 5000
cpu MHz static  : 5000

cpu number      : 5
cpu MHz dynamic : 5000
cpu MHz static  : 5000

cpu number      : 6
cpu MHz dynamic : 5000
cpu MHz static  : 5000

cpu number      : 7
cpu MHz dynamic : 5000
cpu MHz static  : 5000
