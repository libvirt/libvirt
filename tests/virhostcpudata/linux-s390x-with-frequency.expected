CPUs: 8/8, MHz: 5000, Nodes: 1, Sockets: 1, Cores: 8, Threads: 1
